<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseServer</class>
 <widget class="QDialog" name="chooseServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>308</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Port No</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port_text"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ip_address_text"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="server_connect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>186</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string>Start Server</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 233, 79);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Server Details&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_server">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>186</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Stop Server</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
